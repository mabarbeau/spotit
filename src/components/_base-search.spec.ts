import { shallowMount } from '@vue/test-utils'
import BaseSearch from './_base-search.vue'

describe('_base-search.vue', () => {
  it('renders correctly', () => {
    const wrapper = shallowMount(BaseSearch)
    expect(wrapper.html()).toMatchSnapshot()
  })
})
