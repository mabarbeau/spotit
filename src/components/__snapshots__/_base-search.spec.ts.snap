// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`_base-search.vue renders correctly 1`] = `
<v-autocomplete-stub errorcount="1" errormessages="" messages="" rules="" successmessages="" appendicon="$dropdown" backgroundcolor="" hidedetails="true" label="Search" loaderheight="2" clearicon="$clear" flat="true" prependinnericon="mdi-magnify" soloinverted="true" type="text" valuecomparator="function deepEqual(a, b) {
  if (a === b) return true;

  if (a instanceof Date &amp;&amp; b instanceof Date) {
    // If the values are Date, they were convert to timestamp with getTime and compare it
    if (a.getTime() !== b.getTime()) return false;
  }

  if (a !== Object(a) || b !== Object(b)) {
    // If the values aren't objects, they were already checked for equality
    return false;
  }

  var props = Object.keys(a);

  if (props.length !== Object.keys(b).length) {
    // Different number of props, don't bother to check
    return false;
  }

  return props.every(function (p) {
    return deepEqual(a[p], b[p]);
  });
}" nodatatext="$vuetify.noDataText" cacheitems="true" items="" itemcolor="primary" itemdisabled="disabled" itemtext="text" itemvalue="value" menuprops="[object Object]" allowoverflow="true" filter="function _default(item, queryText, itemText) {
        return itemText.toLocaleLowerCase().indexOf(queryText.toLocaleLowerCase()) > -1;
      }" hidenodata="true" class="hidden-sm-and-down"></v-autocomplete-stub>
`;
